testsuite/forge-cli/src/main.rs

```diff
@@ -1693,23 +1693,26 @@ fn realistic_env_max_load_test(
                     (duration.as_secs() / 10).max(60),
                 )
                 .add_system_metrics_threshold(SystemMetricsThreshold::new(
-                    // Check that we don't use more than 14 CPU cores for 30% of the time.
-                    MetricsThreshold::new(14.0, max_cpu_threshold),
+                    // Check that we don't use more than 16 CPU cores for 30% of the time.
```

Comment:
We probably also want to adjust the % of time here.

Reply:
Why adjust the %? Do you have a suggested %?